thirdparty/runtime_environments.go

```diff
@@ -58,3 +66,42 @@ func (c *RuntimeEnvironmentsClient) getImageNames(ctx context.Context) ([]string
 	}
 	return filteredImages, nil
 }
+
+// PackageFilterOptions represents the filtering arguments, each of which is optional.
+type PackageFilterOptions struct {
+	Page    int
+	Limit   int
+	Name    string
+	Manager string
+}
+
+// getPackages returns a list of package changes (name and version) from the corresponding AMI id.
```

Comment:
nit: I don't know if we need to specify name and version here, since that appears to be what the package is